rero_mef/api.py

```
@@ -256,61 +325,85 @@ def replace(self, data, commit=False, dbcommit=False, reindex=False):
-
             except PIDDoesNotExistError:
                 return None
             except NoResultFound:
                 return None
             except OperationalError:
-                get_record_error_count += 1
-                msg = f"Get record OperationalError: {get_record_error_count} {pid}"
-                current_app.logger.error(msg)
+                current_app.logger.error(
+                    f"Get record OperationalError: {attempt + 1} {pid}"
+                )
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)